popSimManu.tex

```diff
@@ -614,9 +614,20 @@ \section*{Discussion}
 history, in large part because of the restriction of constant population sizes.
 An exception is \smcpp, which does not have the same
 restrictions on its inferred history, and as a result does somewhat better
-with the CEU history but tends to overfit the YRI history.  The results for
+with the CEU history but tends to overfit the YRI history.
+The results for
 the two-population \textit{D.~melanogaster} model (Figure~\ref{fig:two_popn_fly})
 are generally similar.
+In these comparisons, \fastsimcoal and \dadi perform almost identically, which is expected because they fit the same models to the same summaries of the data, differing only in how they calculate model expectations and optimize parameters.
+
+All methods for inferring demographic history have strengths and weaknesses (as recently reviewed by \citet{beichman2018review}.)
+We compared inferences from simulated whole genome data, but many factors affect choice of methodology.
+Markovian coalescent methods (\MSMC and \smcpp) require long contiguous stretches of sequence data.
```

Comment:
Actually, 100kb contigs can be enough for psmc.
https://www.biorxiv.org/content/10.1101/382036v1

Reply:
Good point, I wasn't aware of your work on this. But I think we can still call 100 kb contigs "long", certainly relative to RADseq.